README.md

```diff
@@ -184,6 +184,54 @@ public class CatListItemAdapterDelegate extends AbsListItemAdapterDelegate<Cat,
 
 As you see, instead of writing code that casts list item to Cat we can use `AbsListItemAdapterDelegate` to do the same job (by declaring generic types).
 
+## DiffUtil & ListAdapter
+Support library 27.0.1 introduced `ListAdapter` - the new extension of `RecyclerView.Adapter` that uses `AsyncListDiffer` internally. It does calculating diff in the background thread by default and does all particular animations for you items collection. Hence you don't need carry about `notify*` methods, `AsyncListDiffer` does all the job for you. And AdapterDelegates supports it too. 
+
+You have a `DiffItem` interface, that should be implemented by your items. It declares methods for resolving items by some unique ID and by the content. It makes using of `DiffUtil.ItemCallback` super easy.
+
+```java
+public interface DiffItem {
+
+    long getItemId();
+
+    int getItemHash();
+}
+```
+
+ `getItemHash()` is supposed to be implemented through calculation of Object's `hashCode()`. This approach allows us to resolve if content of items has been changed or not. But feel free to make your own implementation
+
+ `getItemId()` is pretty clear and you can use IDs of your items. Obviously it has to be unique. But sometimes when you have some adapter items that don't have any ID (for example footers, headers or section dividers) I suggesting you to use item layout ID. It will look like this:
+
+```java
+class MyFooterItem implements DiffItem {
+    @Override
+    long getItemId() {
+        return (long)R.layout.item_footer
+    }
+ 
+    @Override
+    int getItemHash() {
+        return hashCode()
+    }
+}
+```
+
+If you are using Kotlin you are actually have nothing to do. Kotlin `data class` already has `hashCode()` implementation. It case of Java you will need to generate `equals()` and `hashCode()` methods.
+
+You have a `DiffDelegationAdapter` which is ready to go implementation of `AbsDiffDelegationAdapter` with API rather similar to `ListDelegationAdapter`. 
+
+```java
+public class DiffAdapter extends DiffDelegationAdapter<DiffItem> {
+
+    public DiffAdapter() {
+        delegatesManager.addDelegate(new DiffDogAdapterDelegate());
+        delegatesManager.addDelegate(new DiffCatAdapterDelegate());
+    }
+}
+```
+
+And provide data to it through `setItems()` method. No more carry about which items was deleted, inserted or etc. `AsyncListDiffer` will do all the job for you and will apply all particular animations to your list.
```

Comment:
I would rephrase this a bit: 

You must set data by calling `setItems()`. Internally `AsyncListDiffer` computes and applies all item animations for you.

What do you think about that?

Reply:
Yeah, i'm agree